sql/_32_damson/cbrd_22783_REGEXP_functions/cases/REGEXP_unicode_aware.sql

```diff
@@ -1,17 +1,28 @@
+--REGEXP
 SET NAMES utf8 COLLATE utf8_ko_cs;
 SELECT '가나다라' REGEXP '가';
 SELECT '가나다라' REGEXP '마';
-SELECT REGEXP_REPLACE('가나다라', '[가-나]{4}', '한글');
-SELECT REGEXP_REPLACE('가나다라', '[가-라]{4}', '한글');
 SELECT '가나다' REGEXP '[[:alpha:]]';
 SELECT '123' REGEXP '[[:alpha:]]';
+
+--REGEXP_REPLACE
+SET NAMES utf8 COLLATE utf8_ko_cs;
+SELECT REGEXP_REPLACE('가나다라', '[가-나]{4}', '한글');
+SELECT REGEXP_REPLACE('가나다라', '[가-라]{4}', '한글');
 SELECT REGEXP_REPLACE('가나다라', '[[:alpha:]]', '한글')
 SELECT REGEXP_REPLACE('a1가b2나다라', '[가-다]', '#', 6);
 SELECT REGEXP_REPLACE('a1가b2나다라', '[가-다]', '#', 1, 3);
-SELECT REGEXP_REPLACE (_euckr'가나다라' collate euckr_bin, _utf8' [가-나]{4}', _utf8'다른 문자셋');
-SELECT REGEXP_REPLACE (_euckr'가나다라', _utf8' [가-나]{4}', _utf8'다른 문자셋');
+SELECT REGEXP_REPLACE(_euckr'가나다라' collate euckr_bin, _utf8' [가-나]{4}', _utf8'다른 문자셋');
+SELECT REGEXP_REPLACE(_euckr'가나다라', _utf8' [가-나]{4}', _utf8'다른 문자셋');
 
 SET NAMES utf8; 
-SELECT REGEXP_REPLACE (_euckr'가나다라', _utf8' [가-나]{4}', _utf8'다른 문자셋');
-SELECT REGEXP_REPLACE ('가나다라' COLLATE utf8_ko_cs,  _utf8' [த]' COLLATE utf8_tr_cs, 'INVALID');
+SELECT REGEXP_REPLACE(_euckr'가나다라', _utf8' [가-나]{4}', _utf8'다른 문자셋');
+SELECT REGEXP_REPLACE('가나다라' COLLATE utf8_ko_cs,  _utf8' [த]' COLLATE utf8_tr_cs, 'INVALID');
 
+--REGEXP_SUBSTR
+SET NAMES utf8 COLLATE utf8_ko_cs;
+SELECT REGEXP_SUBSTR('삼성로 86길, 강남구, 서울특별시', ',[^,]+,', 1, 1);
+SELECT REGEXP_SUBSTR('삼성로 86길, 강남구, 서울특별시', '[[:alpha:]]+', 1, 3);
+SELECT REGEXP_SUBSTR('11억 8112만 5400원', '\d+[[:alpha:]]', 3, 1);
+SELECT REGEXP_SUBSTR(_euckr'가나다라' COLLATE euckr_bin, _utf8' [가-나]{4}' COLLATE utf8_ko_cs);
+SELECT REGEXP_SUBSTR('가나다라' COLLATE utf8_ko_cs,  _utf8' [த]' COLLATE utf8_tr_cs);
```

Comment:
It would better add a new file for each function

Reply:
Now files are divided by Unicode related parts cases and functional cases. How is it more useful to be dealt with?

Reply:
referring the SQL guide doc in cubrid-testtools;
> Don't write too much queries in one test case file, it's hard to verify when the case fails. You can separate them into several files.

I don't think Unicode related test cases should be written in a single file.